### PR TITLE
Realistic names for new assets from Tanks DLC

### PR DESCRIPTION
**Pull request by xrufix**
**When merged this pull request will:**
Add realistic names for new assets from Tanks DLC.

Vehicles:
- [X]  LSV AT-Variants.
- [X] Rhino MGS -> Rooikat
- [X] T140 Angara -> T14 Armata
- [X] AWC 302 Nyx -> Wiesel 2;
    - Wiesel 2 Ozelot (AA)
    - Wiesel 2 ATGM
    - Wiesel 2 AFF
    - Wiesel 2 MK20
- [x]  Zamak MLR -> Kamaz MLR
- [x] MB4WD armed -> Jeep Wrangler
- [x] New Marid -> Otokar ARMA

Vehicle Weapons:
- [X] Vorona -> Metis-M
- [X] Rooikat 120mm Gun -> GT12
- [x] Rooikat Coax SPMG .338 ->Denel MG4 (Added manufacturer's name to distinguish from H&K MG4)
- [x] Armata Commander M2 -> KORD
- [x] Armata K Commander Autocannon -> 2A42
- [x] Armata K Main Cannon -> 2A82-1M (source: [German Wikipedia](https://de.wikipedia.org/wiki/T-14#Bewaffnung))
- [x] Ozelot SAAMI -> FIM92F Stinger
- [x] Wiesel MK20 PKT -> MG3
- [x] Wiesel MK20 Autocannon -> MK20 Rh 202
- [x] Wiesel AT Firefist -> Keep for now

Weapons:
- [X] Vorona -> Metis-M
- [X] MAAWS Mk4 Mod 0 / Mod 1 -> no change

I was in the flow, so I added Optics Attachments as well.

More to be added. Please add assets i may have missed in this PR.

### CODE REVIEW

**Comment by xrufix** [comment]
Ah, missed changing the name, thanks!

File: addons/realisticnames/CfgWeapons.hpp [hunk 1]
```diff
@@ -541,6 +545,9 @@ class CfgWeapons {
     class ACE_LMG_coax_L94A1_mem3: LMG_coax {
         displayName = "L94A1";
     };
+    class ACE_LMG_coax_ext_MG3: LMG_coax {
```
Comment (PabstMirror) [inline addons/realisticnames/CfgWeapons.hpp]: inherit from `LMG_coax_ext`

Comment (xrufix) [reply addons/realisticnames/CfgWeapons.hpp]: thx, fixed

File: addons/vehicles/CfgVehicles.hpp [hunk 2]
```diff
@@ -229,4 +229,27 @@ class CfgVehicles {
...
+        class Turrets : Turrets {
+            class MainTurret : MainTurret {};
+        };
+    };
+    class LT_01_cannon_base_F : LT_01_base_F {
+        class Turrets : Turrets {
+            class MainTurret : MainTurret {};
+        };
+    };
+    class I_LT_01_cannon_F : LT_01_cannon_base_F {
```
Comment (xrufix) [inline addons/vehicles/CfgVehicles.hpp]: Can this be done with less repetitive code? I guess I still haven't fully understood config inheritance...

Comment (jonpas) [reply addons/vehicles/CfgVehicles.hpp]: @commy2

**Comment by severgun** [comment]
>  MAAWS -> ?

https://saab.com/land/weapon-systems/support-weapons/carl-gustaf-m4/

**Comment by ninelore** [comment]
The Rhino MGS 120mm Turret is called Falcon Autoloader Turret

**Comment by BaerMitUmlaut** [comment]
No, it's not a Falcon turret. It's a turret made specifically for the Rooikat 120 and uses a Denel GT12 120mm smooth bore gun.

**Comment by xrufix** [comment]
@severgun I know that the Armata doesn't have a autocannon. But it would be nice to at least give it a name that's realistic, even if it isn't real.

Thanks on the other info though.

**Comment by xrufix** [comment]
Regarding the MAAWS: I know it's a Carl Gustaf (although i would have guessed it's an M3E1, not M4, but let's keep that aside for a moment). I wonder: should we name it Carl Gustaf or MAAWS? MAAWS is the US designation for the same weapon.

File: addons/vehicles/CfgVehicles.hpp (hunk 2 above)
Comment (commy2) [reply addons/vehicles/CfgVehicles.hpp]: Why not add the weapon to LT_01_cannon_base_F and ignore I_LT_01_cannon_F ?

**Comment by xrufix** [comment]
Regarding the Vorona: It's designated 9M135. Kornet is 9M133. Metis-M is 9M131. But visually it nearly identical to the Metis-M. I think it's a direct copy.

**Comment by Drofseh** [comment]
> Carl Gustaf or MAAWS

Given that the side is called NATO I would call it the Carl Gustaf

File: addons/vehicles/CfgVehicles.hpp (hunk 2 above)
Comment (xrufix) [reply addons/vehicles/CfgVehicles.hpp]: Because it doesn't work. That would have been my first guess.

**Comment by xrufix** [comment]
@Drofseh It's not even exclusive to NATO.

**Comment by xrufix** [comment]
Still, then the question remains: How to distinguish the CG with optical sight from the one with digital sight?

Keep Vanillas "Mod. 0" and "Mod. 1"?

**Comment by Blutze** [comment]
`<English>Nightforce NXS (Grenn Hex)</English>`
Pretty sure this should be "Green".

**Comment by xrufix** [comment]
Fixed the typo (with a typo in the commit message, lol).

**Comment by severgun** [comment]
>  Rooikat Coax SPMG .338 ->?
Eng wiki say MG4 (M1919A4)

**Comment by severgun** [comment]
>  Wiesel AT Firefist -> ?
German wiki

TOW/TOW2

**Comment by xrufix** [comment]
@severgun regarding Firefist: TOWs are wire guided, the Firefist obviously isn't and even has top-attack-mode.

Gona change the SPMG->M4 now.

**Comment by xrufix** [comment]
I am not going to change anything on the "Firefist" for now, as like I stated above, I do not think it should be named TOW, except if guidance etc. are also changed. But I doubt this would be a good idea.

PR is ready for review and no longer WIP.

File: addons/realisticnames/CfgVehicles.hpp [hunk 3]
```diff
@@ -186,6 +186,13 @@ class CfgVehicles {
     class O_APC_Wheeled_02_base_F : APC_Wheeled_02_base_F {
         displayName = CSTRING(APC_Wheeled_02_rcws_Name);
     };
+    class APC_Wheeled_02_base_v2_F : APC_Wheeled_02_base_F{};
```
Comment (BaerMitUmlaut) [inline addons/realisticnames/CfgVehicles.hpp]: Why the inherition here?

Comment (xrufix) [reply addons/realisticnames/CfgVehicles.hpp]: Why not?

**Comment by madpat3** [comment]
speaking of the "Firefist" of the Wiesel, the model of the rocket looks like a Javelin:
https://www.google.de/search?q=anti+tank+missile&source=lnms&tbm=isch&sa=X&ved=0ahUKEwiRn_alpMPaAhVEEVAKHXx7C6QQ_AUICigB&biw=1920&bih=1012#imgrc=TE3KyvvynDjgTM:

**Comment by madpat3** [comment]
it seems that there're existing many different systems, to launch a javeline missile from a vehicle. 
sometimes they just take a footmobilesystem and "tie" it to the turret:)
2l-image-60

**Comment by xrufix** [comment]
IMHO, it looks more like a Spike (the same used on other Vanilla turrets):

**Comment by madpat3** [comment]
yes, and maybe the same exsists for javeline rockets, or will be, for the game is futuristic.

**Comment by BaerMitUmlaut** [comment]
I think this should be decided based on the capabilites of the rocket. It's not a TOW since it can do a top down attack. Can it lock onto lasers? Then it's not a Javelin either.

**Comment by PabstMirror** [comment]
TOW-2B can do top attack

**Comment by BaerMitUmlaut** [comment]
Nope, the TOW-2B does an overfly top attack, not a top down attack like the Javelin uses. It can be seen here:
https://www.youtube.com/watch?v=9IjAPVTtYDE

**Comment by madpat3** [comment]
> I think this should be decided based on the capabilites of the rocket. It's not a TOW since it can do a top down attack. Can it lock onto lasers? Then it's not a Javelin either.

just tested it. it cannot lock onto laser.
i wouldn't make the name depend on the launcherplatform.
for example the agm 84 is normally launched from fighterplanes, but can also be launched from the ground, where they use a certain launchplattform with a booster (A/B44G-2).
but also the ground launched rockets are called AGM 84 and not A/B44G-2 or something.

**Comment by bux** [comment]
> please don't missunderstand my interference in your discussion. I'm just very interested in this realism thing and hope i can help a little.
> But if you don't want any externals in here, i will immediately leave this thread.

No worries. That‘s why this discussion is on github. 😚

**Comment by madpat3** [comment]
i'd say it's a javeline, although the monitor, where the gunner locks onto target is different from the titan launcher. but the rocket looks like a javeline and it behaves like one. only fire and forget.

unlike the spike. the rocket looks different
6n2

and behaves different. it can be use as a fire and forget weapon and also as fire and steer, like a wire guided weapon.

**Comment by madpat3** [comment]
speaking of titan launcher. why not name it Mini Spike?
http://defense-update.com/photos/mini_spike.html

**Comment by xrufix** [comment]
@madpat3 Someone had that [idea earlier](https://github.com/acemod/ACE3/blob/master/addons/realisticnames/stringtable.xml#L1586-L1601). But someone decided it should be left [commented out](https://github.com/acemod/ACE3/blob/master/addons/realisticnames/CfgWeapons.hpp#L184-L189).

**Comment by TheNigal** [comment]
Would the MAAWS not be the Carl Gustav instead? That is the name of the weapon system, and most other name changes have used the manufacturer name instead of the military designation.

File: addons/vehicles/CfgVehicles.hpp (hunk 2 above)
Comment (PabstMirror) [reply addons/vehicles/CfgVehicles.hpp]: Looking at base configs, it was possible.
I double checked in-game and it's working

**Comment by PabstMirror** [comment]
CUP calls theirs: MAAWS
RHS calls theirs: M3 MAAWS
So it would be consistent with the community (and their magazines wells?)
But on the other hand, cg is it's actual name....

I'm going to merge this as is, but we can still think about changing it

**Comment by madpat3** [comment]
i would call the AWC 302 Nyx Wiesel 3, for it looks more like a successor of the todays' Wiesel 2.

**Comment by madpat3** [comment]
Leopard 2SG looks more like the Leopard 2 Revolution.
Although they look very similar, due to the new Rheinmetall-Armor both tanks are equipped with, they are not equal (espacially how the headlights are integrated and the type of secondary armament on top of the turret).

Leopard 2SG:
st_20150629_xmobile29_1457231

Leopard 2 Revolution:
das-ist-der-leopard-2-revolution-1199x800-9f5b484f18e44f70

**Comment by bux** [comment]
@madpat3 please don't revive old issues or PRs. Feel free to open a new one.

**Pull request merged**